scipy/stats/tests/test_qmc.py

```diff
@@ -533,8 +535,12 @@ def test_continuing(self, scramble):
         assert_allclose(sample, ref_sample[n_half:], atol=1e-1)
 
     @pytest.mark.parametrize("scramble", scramble, ids=ids)
-    def test_reset(self, scramble):
-        engine = self.engine(d=2, scramble=scramble)
+    @pytest.mark.parametrize(
+        "seed",
+        (170382760648021597650530316304495310428, None),
```

Comment:
We're still missing a test of `reset()` when `seed` is a live `Generator` instance.